install-deps.sh

```diff
@@ -374,7 +374,6 @@ else
 	if [ -n "$dts_ver" ]; then
             ensure_decent_gcc_on_rh $dts_ver
 	fi
-        ! grep -q -i error: $DIR/yum-builddep.out || exit 1
```

Comment:
can we move this line before the `if [ -n "$dts_ver" ]; then` block?

Reply:
In my environment `ensure_decent_gcc_on_rh` does barely anything (devtoolset 8 is pre-installed by Dockerfile, and it's not 8.3.1). I'll check it, but don't think it's going to change much :S